src/backend/tests/models/charge_types/test_unclassified.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+from datetime import datetime, timedelta
+
+from expungeservice.models.charge_types.unclassified_charge import UnclassifiedCharge
+from expungeservice.models.expungement_result import EligibilityStatus
+from expungeservice.models.disposition import Disposition
+
+from tests.factories.charge_factory import ChargeFactory
+from tests.models.test_charge import ChargeTypeTestsParent
+
+
+class TestSingleChargeUnclassified(ChargeTypeTestsParent):
+    def setUp(self):
+        ChargeTypeTestsParent.setUp(self)
```

Comment:
```suggestion
        super().setUp()
```

See https://stackoverflow.com/q/576169/2750819